requirements/libraries.txt

```diff
@@ -2,49 +2,43 @@
 backports.csv==1.0.7
 base36==0.1.1
 beautifulsoup4==4.8.2
-boto3==1.7.80
+boto3==1.22.7
 dj-database-url==0.5.0
-djangorestframework==3.11.2
-django-cors-headers==3.9.0
+django-cors-headers==3.11.0
 django-csp==3.7
-django-elasticsearch-dsl==7.1.4
-django-extensions==2.1.3
-django-flags==5.0.8
+django-debug-toolbar==3.4.0
```

Comment:
We do need this in `libraries.txt`? It *should* only be used for local dev, AFAIK.

Reply:
Local dev. Good catch.